src/app/login/login.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="Login-background">
+  <div [formGroup]="formGroup" class="Login-container">
+    <h1><strong>Login</strong></h1>
+
+    <mat-form-field class="mat-Input">
+      <mat-label>Username</mat-label>
+      <input matInput type="text" placeholder="username" formControlName="username" />
+      <mat-error *ngIf="formGroup.controls['username'].invalid">
+        Username is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+    <mat-form-field class="mat-Input">
+      <mat-label>Password</mat-label>
+      <input matInput [type]="showPassword ? 'text' : 'password'" placeholder="password" formControlName="password" />
+      <mat-icon matSuffix (click)="togglePasswordVisibility()">
+        {{showPassword ? 'visibility_off' : 'visibility'}}
+      </mat-icon>
+      <mat-error *ngIf="formGroup.controls['password'].invalid">
+        Password is <strong>required</strong>
+      </mat-error>
+    </mat-form-field>
+
+      <button mat-button class="mat-Button" color="custom" (click)="login()">
+        Login
+      </button>
```

Comment:
Вместо ручного чека в методе login:
https://github.com/Chevich123/learn-fe/blob/81c33103a3c9ddb4770ea680ab1c22df9a8e8c7a/src/app/login/login.component.ts#L31-L33

Можно добавить кнопке проп:
```suggestion
      <button [disabled]="formGroup.invalid" mat-button class="mat-Button" color="custom" (click)="login()">
        Login
      </button>
```

тогда по хорошему надо дописать стиль `.mat-Button:disabled`, чтобы было визуально понятно, что кнопка некликабельна